runtime/oti/j9nonbuilder.h

```diff
@@ -42,8 +42,9 @@
 #define J9VM_ASYNC_MAX_HANDLERS 32
 
 #define CLASSNAME_INVALID			0
```

Comment:
Please add a bit of doc here saying these are bit values.

Reply:
Add comments as suggested above.